setup-build-freetype.py

```diff
@@ -81,8 +81,10 @@
 if "linux" in sys.platform:
     c_flags = cxx_flags = "-O2"
     ld_flags = ""
-    if ("arm" not in platform.machine() and 
-            platform.machine() != "aarch64"):
+    if (
+        platform.machine == "x86_64"
+        or platform.machine == "x86"
+    ):
```

Comment:
No need for "x86"...

Reply:
Ok can change that. I added the second case if the script is executed on an 32bit machine... Don't know if the output of `machine` is there also `x86_64`.

By the way, I missed also the `()` after the `machine` function call.